### PR TITLE
Create jobs project

### PR DESCRIPTION
**Pull request by robinoconnor**

### CODE REVIEW

File: src/SFA.DAS.EmployerFinance/SFA.DAS.EmployerFinance.csproj [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance/SFA.DAS.EmployerFinance.csproj]: I've committed this to the branch as realised we have missed it out

File: src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <ApplicationIcon />
+    <OutputType>Exe</OutputType>
+    <StartupObject />
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Remove="appsettings.Development.json" />
+    <None Remove="appsettings.json" />
+    <None Remove="NLog.config" />
```
Comment (elliotritchie) [inline src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj]: The file name is lowercase so this results in two files appearing in the project (at least in Rider). Can it be renamed in the csproj to `nlog.config`?

File: src/SFA.DAS.EmployerFinance/SFA.DAS.EmployerFinance.csproj (hunk 1 above)
Comment (elliotritchie) [inline src/SFA.DAS.EmployerFinance/SFA.DAS.EmployerFinance.csproj]: Can `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` be added?

File: src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj [hunk 3]
```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <ApplicationIcon />
+    <OutputType>Exe</OutputType>
+    <StartupObject />
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
```
Comment (elliotritchie) [inline src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj]: Can `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` be added?

File: src/SFA.DAS.EmployerFinance.Jobs/NServiceBusStartup.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+            EmployerFinanceConfiguration employerFinanceConfiguration)
+        {
+            _container = container;
+            _environmentService = environmentService;
+            _employerFinanceConfiguration = employerFinanceConfiguration;
+        }
+        
+        public async Task StartAsync()
+        {
+            var endpointConfiguration = new EndpointConfiguration("SFA.DAS.ProviderRelationships.Jobs")
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Jobs/NServiceBusStartup.cs]: sure

File: src/SFA.DAS.EmployerFinance.Jobs/Program.cs [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                        builder.AddJsonFile(AppSettingFilePath);
+                    })
+                    .ConfigureLogging(builder => 
+                    { 
+                        builder.SetMinimumLevel(LogLevel.Debug)
+                               .AddNLog();
+                    })
+                    .ConfigureServices(collection =>
+                    {
+                        //Wire up the IOC container to the built in web job IOC container
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Jobs/Program.cs]: Ah I put that in there to show that this is what this code was doing as its not completely clear to a new dev. I can remove this however if you think its not needed?

[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Jobs/Program.cs]: i'll remove this for now as we can always put it back in if people need it later on

File: src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj [hunk 6]
```diff
@@ -0,0 +1,52 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <ApplicationIcon />
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj]: nope, removed

File: src/SFA.DAS.EmployerFinance.Host/appsettings.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+﻿{
```
[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Host/appsettings.json]: The previous host project have had web.config files so i've assumed the app settings file may be needed but i'll check with Ewan

[the remark this answers is not in the document]
Comment (robinoconnor) [reply src/SFA.DAS.EmployerFinance.Host/appsettings.json]: This has been removed as its not needed

**Pull request merged**